.coveragerc

```diff
@@ -0,0 +1,12 @@
+[run]
+data_file = .coverage_mqt  # to avoid being overridden by server tests
+omit = *system_site_packages*
+
+[report]
+# Regexes for lines to exclude from consideration
+exclude_lines =
+    # Have to re-enable the standard pragma
+    pragma: no cover
+
+    # Don't complain if non-runnable code isn't run:
+    if __name__ == .__main__.:
```

Comment:
Anyone know why this isn't being excluded? https://coveralls.io/builds/3173483/source?filename=travis%2Fgetaddons.py#L77